### ISSUE TITLE
[BUG] Setting ports for container throws the error: is not a valid port definition

### ISSUE THREAD

**Issue by MurzNN** [saltstack-formulas/docker-formula#279]
I'm trying to set ports mapping for the contaner, using this pillar structure:
```yaml
docker:
  wanted:
    - docker
    - compose

  containers:
    running:
      - pgadmin

    pgadmin:
      image: dpage/pgadmin4
      ports:
        - '88:80'
      volumes:
        - /pgadmin4:/var/lib/pgadmin
```
The format of `ports` line I get from [pillar.example](https://github.com/saltstack-formulas/docker-formula/blob/master/pillar.example) file.

But when applying this config, I got the error:
```
          ID: docker-containers-pgadmin-running
    Function: docker_container.running
        Name: pgadmin
      Result: False
     Comment: Failed to translate input. Additional info follows:
              
              invalid:
                  ----------
                  ports:
                      '{88: 80}' is not a valid port definition
              
              If you feel this information is incorrect, the skip_translate argument can be used to skip input translation for the argument(s) identified as invalid. See the documentation for details.
     Started: 13:59:50.387972
    Duration: 33.969 ms
     Changes:   
```

Where can be the problem? Maybe the pillar.example file is outdated, and we need to use other format for ports settings?

**Comment by MurzNN** [comment]
When analyzing sources, I figured-out other way to define ports, that works well:
```yaml
      port_bindings:
        - '88:80'
```
Should I update the `pillar.example` to add this example, or it is wrong?

**Comment by noelmcloughlin** [comment]
Hey Aexey,
Yes, please do that update.
regards
Noel

**Comment by MurzNN** [comment]
Here is PR https://github.com/saltstack-formulas/docker-formula/pull/280

**Comment by MurzNN** [comment]
Resolved via #280
